test/re/frida/MethodTest.java

```diff
@@ -55,14 +59,183 @@ public void replacementPropagatesExceptions() {
         thrown.expectMessage("Already dead");
         badger.die();
     }
+    
+    @Test
+    public void interfaceCannotBeInstantiated() {
+        loadScript("var X509TrustManager = Java.use('javax.net.ssl.X509TrustManager');" +
+                "try {" +
+                "  var tm = X509TrustManager.$new();" +
+                "  send('ok');" +
+                "} catch (e) {" + 
+                "  send('couldnt create trustmanager');" + 
+                "}"
+                );
+        assertEquals("ok", script.getNextMessage());
+    }
+    
+    @Test
+    public void genericReturnJavaLangClass() {
+        loadScript("var c = Java.use ('java.lang.Class');" +
```

Comment:
Extra space after `use`.

(Also see other comment about uppercase-camel naming of class variables.)